src/replit/database/default_db.py

```diff
@@ -1,13 +1,38 @@
 """A module containing the default database."""
-from os import environ
+from os import environ, path
 from typing import Optional
+import threading
 
 from .database import Database
 
-db: Optional[Database]
-db_url = environ.get("REPLIT_DB_URL")
-if db_url:
-    db = Database(db_url)
-else:
-    # The user will see errors if they try to use the database.
-    db = None
+db: Optional[Database] = None
+
+db_url: str = environ.get("REPLIT_DB_URL")
+
+
+def reload_db() -> None:
+    """
+    Reloads the database. The database token expires every 20h.
+    """
+    global db
+    global db_url
+    if path.exists("/tmp/replitdb"):
+        with open("/tmp/replitdb", 'r') as file:
+            db_url = file.read()
+    else:
+        db_url = environ.get("REPLIT_DB_URL")
+
+    if db_url:
+        db = Database(db_url)
+    else:
+        # The user will see errors if they try to use the database.
+        db = None
```

Comment:
Sorry I missed this. But there's a problem here. The global `db` being updated will not be updated for callers that are importing it via `from default_db import db`. I suggest wrapping it in a `get_db()` function and having the callers call that.

Reply:
Might be doable to periodically update the db_url https://github.com/replit/replit-py/blob/master/src/replit/database/database.py#L419

Reply:
Nice catch! i think updating the db_url property should work here, let me take  a look